ckks/marshaler.go

```diff
@@ -3,24 +3,19 @@ package ckks
 import (
```

Comment:
Since marshalers are well modularized now, I think there is no point in keeping `ckks/marshaler.go`.  The methods could be moved to `ckks/ciphertext.go`.

Reply:
fixed